src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1063,7 +1070,8 @@ private static VersionInfo GetVersionInfo()
                 // verWrittenCur: 0x00010009, // Introduced _flags
                 //verWrittenCur: 0x0001000A, // Added ForceVector in Range
                 //verWrittenCur: 0x0001000B, // Header now retained if used and present
-                verWrittenCur: 0x0001000C, // Removed Min and Contiguous from KeyType, and added ReadMultilines flag to OptionFlags
+                //verWrittenCur: 0x0001000C, // Removed Min and Contiguous from KeyType, and added ReadMultilines flag to OptionFlags
+                verWrittenCur: 0x0001000D, // Added decimal marker option to allow for ',' to be a decimal marker
```

Comment:
Just a heads-up that I'm also introducing a new option and updating the verWrittenCur of TextLoader on my PR here:
https://github.com/dotnet/machinelearning/pull/5147

So I guess we'll have some merge conflicts with our PRs 😱 😛